data/themes/darktable.css

```diff
@@ -1148,9 +1164,13 @@ dialog scrollbar
    min-width: 180px;
 }
 
+#preferences_notebook combobox menuitem
+{
+  padding: 0.2em;
+}
+
 #preferences_notebook button
 {
-  margin: 0 2px;
   padding: 0 2px;
```

Comment:
Just made some test and I think it would be better with margin like that:
`
margin-left: 1px;`